addons/stock_account/models/stock_location.py

```diff
@@ -30,3 +30,13 @@ def _should_be_valued(self):
         if self.usage == 'internal' or (self.usage == 'transit' and self.company_id):
             return True
         return False
+
+    @api.onchange('usage')
```

Comment:
field should be compute, store=True and this should be a compute not an onchange (should follow the same behavior backend/frontend)

Reply:
@amoyaux okay I updating this one but it's mins that now we define compute with store true instead of on change?

if yes, then if there is any condition when we define compute or on change plz share it so we consider this in the future PRs.